docs/usage/cli/filter.rst

```diff
@@ -63,3 +71,184 @@ The filter command allows you to select a specific number of sequences from spec
     --output filtered.fasta
 
 This subsampling and filtering will reduce the number of sequences in the tutorial data set from 34 to 24.
+
+Subsampling beyond the capabilities of ``augur filter``
+-------------------------------------------------------
+
+There are some subsampling strategies in which a single call to ``augur filter``
+does not suffice. One such strategy is "tiered subsampling". In this strategy,
+mutually exclusive sets of filters, each representing a "tier", are sampled with
+different subsampling rules. This is commonly used to create geographic tiers.
+Consider this subsampling scheme:
+
+    Sample 20 sequences from North America and 50 sequences from Europe.
+
+This cannot be done in a single call to ``augur filter``. Instead, it can be
+decomposed into multiple schemes, each handled by a single call to ``augur
+filter``. Additionally, there is an extra step to combine the intermediate
+samples.
+
+    1. Sample 20 sequences from North America.
+    2. Sample 50 sequences from Europe.
+    3. Combine the samples.
+
+Calling ``augur filter`` multiple times
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create each intermediate sample using ``augur filter``, then join them together
+to create the final subsample.
+
+.. code-block:: bash
+
+    # 1. Sample 20 sequences from North America
+	augur filter \
+      --sequences sequences.fasta \
+      --metadata metadata.tsv \
+      --query "region == 'North America'" \
+      --subsample-max-sequences 20 \
+      --output-sequences north_america_sample.fasta \
+      --output-metadata north_america_sample.tsv
+
+    # 2. Sample 50 sequences from Europe
+	augur filter \
+      --sequences sequences.fasta \
+      --metadata metadata.tsv \
+      --query "region == 'Europe'" \
+      --subsample-max-sequences 50 \
+      --output-sequences europe_sample.fasta \
+      --output-metadata europe_sample.tsv
+
+    # 3.1 Combine FASTA files
```

Comment:
I think the more canonical way is to use another `augur filter` command here, as you've used below. This resolves duplicates, which is a very common situation in my experience. I would drop this first example and simply use the  "`--output-strains` x n  + `--exclude-all --include ...`" example.

You can then drop the "Differences:" section entirely. You may want to add a note/discussion about not using "--sequences" when not necessary for performance reasons (or does `augur filter` conditionally read these anyway?)

Reply:
Done in 9357a3be with a note on resolving duplicates.

> does `augur filter` conditionally read these anyway?

Sometimes. If the input `--sequences` is specified, it will be read to check that the metadata and sequences are paired. The special case here is that `--exclude-all` [bypasses](https://github.com/nextstrain/augur/blob/7d75fb19941785c5e660f4376afd1db77badc6dd/augur/filter/_run.py#L44-L45) that behavior.

I've dropped the mention of performance benefits entirely - the example as-is works fine and is optimal with `--exclude-all`. This note distracts from the purpose of the guide.